03_Collection Types.playground/section-1.swift

```diff
@@ -46,6 +47,10 @@ var array = [Int](count: 3, repeatedValue: 0) //Initalizes an array of lenght 3
 
 var compundArray = array + emptyArray
 
+var reverseShoppingList: [String] = shoppingList.reverse()
+
+reverseShoppingList.removeLast() // Remove last item, the other way with removeFirst()
+reverseShoppingList.popLast() // Optional remove last item so that won't crash when array is empty, don't have popFirst()
```

Comment:
@Tj3n can you further explain a use case for `remove last item so that won't crash when array is empty`?